cmd/juju/application/setseries.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package application
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"github.com/juju/utils/series"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/application"
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/juju/block"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+// NewSetSeriesCommand returns a command which updates the series of
+// an application.
+func NewSetSeriesCommand() cmd.Command {
+	return modelcmd.Wrap(&setSeriesCommand{})
+}
+
+// setSeriesAPI defines a subset of the application facade, as required
+// by the set-series command.
+type setSeriesAPI interface {
+	BestAPIVersion() int
+	Close() error
+	UpdateApplicationSeries(string, string, bool) error
+}
+
+// setSeriesCommand is responsible for updating the series of an application or machine.
+type setSeriesCommand struct {
+	modelcmd.ModelCommandBase
+	modelcmd.IAASOnlyCommand
+
+	setSeriesClient setSeriesAPI
+
+	applicationName string
+	force           bool
+	series          string
+}
+
+var setSeriesDoc = `
+When no flags are set, an application series value with be set within juju.
```

Comment:
> When no flags are set, an application series value with be set within juju.

Should that be "When no flags are set, an application series value **will** be set within juju."

Reply:
updated